packages/network-controller/src/rpc-service/rpc-service.test.ts

```diff
@@ -191,7 +379,7 @@ describe('RpcService', () => {
         const jsonRpcRequest = {
           id: 1,
           jsonrpc: '2.0' as const,
-          method: 'eth_chainId',
+          method: 'eth_unknownMethod',
```

Comment:
I discovered while making these changes that these tests were actually incorrect.